yarn-project/prover-client/src/orchestrator/orchestrator.ts

```diff
@@ -61,6 +63,8 @@ import {
 import { type MergeRollupInputData, ProvingState, type TreeSnapshots } from './proving-state.js';
 import { TX_PROVING_CODE, TxProvingState } from './tx-proving-state.js';
 
+const AGGREGATION_OBJECT_LEN = 16; // fields
```

Comment:
Lets move this and the other one in `bb_prover` to noir contants or something.

Reply:
Moved to noir constants and it's now used in the rollup contract as well.